src/modules/dashboard/admin/adminHandlers/arenaAdminHandlers.ts

```diff
@@ -0,0 +1,27 @@
+import type { Lifecycle } from '@hapi/hapi';
+
+import { ArenaPlayer, Item } from '../../../../models';
+import { ArenaGame, findActiveArenaGame } from '../../../../models/ArenaGame';
+
+export const getCurrentArenaGameState: Lifecycle.Method = async (_request, h) => {
+  const activeGame = await findActiveArenaGame();
+  await activeGame?.reload({
+    include: {
+      model: ArenaGame,
+      include: [
+        {
+          model: ArenaPlayer,
+          include: [
+            {
+              association: ArenaPlayer.associations._weapons,
+              include: [Item.associations._weapon, Item.associations._traits],
+              as: '_weapons',
+            },
+          ],
+        },
+      ],
+    },
+  });
+
+  return h.response({ arenaGame: activeGame }).code(200);
+};
```

Comment:
separated handler files according to section. Ex: arenaAdmin Handlers , towerAdminHandlers, etc